CHANGELOG.md

```diff
@@ -11,6 +11,11 @@
 ### Added
 
 * Packages - dev & tests
+* Support for audio loader and decoder, which uses libsndfile library to decode wav files
```

Comment:
please modify change log to what is applicable to this PR incrementally from others